c_sharp/Snpe_EP/vgg16_image_classification/README.md

```diff
@@ -0,0 +1,82 @@
+# Image classification with VGG16 in C# using SNPE Execution Provider
+1.  This image classification sample uses the VGG16 model from [SNPE turtorial](https://developer.qualcomm.com/sites/default/files/docs/snpe/tutorial_onnx.html) with DLC format. Wrap it as a custom node in an ONNX model to inference with SNPE Execution Provider.
+
+2.  The sample uses the Onnxruntime SNPE Execution Provider to run inference on various Qualcomm devices like Qualcomm CPU, DSP, etc. It supports Windows ARM64.
+
+# Prerequisites
+1. Setup a Linux environment by [WSL2](https://learn.microsoft.com/en-us/windows/wsl/)
+2. Download SNPE SDK from Qualcomm's developer site [here](https://developer.qualcomm.com/software/qualcomm-neural-processing-sdk)

```

Comment:
it should be https://developer.qualcomm.com/software/qualcomm-neural-processing-sdk/windows-on-snapdragon
and we should mention that the developer needs to request access to it.